common4j/src/main/com/microsoft/identity/common/java/platform/Device.java

```diff
@@ -89,7 +89,7 @@ public static void setIsInPersonalProfileButClouddpcWorkProfileAvailable(final b
     }
 
     @GuardedBy("sLock")
-    public static boolean isInPersonalProfileButClouddpcWorkProfileAvailable() {
+    public static Boolean isInPersonalProfileButClouddpcWorkProfileAvailable() {
```

Comment:
>isInPersonalProfileButClouddpcWorkProfileAvailable [](http://example.com/codeflow?start=26&length=50)

who's caller? Can it handle null?

Reply:
Caller can handle null, issue was that method return signature of `isInPersonalProfileButClouddpcWorkProfileAvailable` used `boolean` type, which cannot be `null`

Reply:
Caller is in `MicrosoftAuthorizationRequest` btw